vm/src/import.rs

```diff
@@ -83,25 +81,30 @@ pub fn import_file(
     file_path: String,
     content: String,
 ) -> PyResult {
-    let sys_modules = vm.get_attribute(vm.sys_module.clone(), "modules").unwrap();
-    let code_obj = vm
-        .compile(&content, &compile::Mode::Exec, file_path.clone())
+    let code_obj = compile::compile(&content, &compile::Mode::Exec, file_path)
         .map_err(|err| vm.new_syntax_error(&err))?;
-    // trace!("Code object: {:?}", code_obj);
+    import_codeobj(vm, module_name, code_obj)
+}
 
+pub fn import_codeobj(vm: &VirtualMachine, module_name: &str, code_obj: CodeObject) -> PyResult {
     let attrs = vm.ctx.new_dict();
     attrs.set_item("__name__", vm.new_str(module_name.to_string()), vm)?;
+    let file_path = &code_obj.source_path;
     if !file_path.starts_with("frozen") {
         // TODO: Should be removed after precompiling frozen modules.
```

Comment:
The `TODO` should probably be removed if  we keep the way we find this is a frozen module the same.

Reply:
What should the behavior be for this? Only set __file__ if not a frozen module? Cause doing `frozen *` as the module name does seem sort of hacky.

Reply:
Yes. Frozen modules should not have a `__file__` attribute. This can be solved in a later stage but the `TODO` should be changed.

Reply:
I just changed CodeObject.source_path to an `Option` in another branch and I'm going to open a PR for it once this merges.